[Consul] CTS lab (#311)
hashicorp/katakoda: consul-terraform-sync-intro/step4.md

After checking the configuration for the tasks, you can run Consul-Terraform-Sync in daemon mode, which is the default mode. In this mode, Consul-Terraform-Sync passes through a 
once-mode phase. 

The once-mode phase will run all the tasks once and then becomes a long running process. 

#### Start Consul-Terraform-Sync in daemon mode

Move to the terminal dedicated to Consul-Terraform-Sync and set
the token as an environment variable.

`source consul_env.conf`{{execute T2}}

Once the environment variables are set, you can run Consul-Terraform-Sync.

`consul-terraform-sync -config-file cts-config.hcl`{{execute T2}}

After startup, Consul-Terraform-Sync will:

1. run Terraform 
1. create a folder, named `sync-tasks`

>  
> The folder will be created inside the `working_directory` defined in the configuration for each task. If no working directory has been configured Terraform will create the 
> folder in the directory where Consul-Terraform-Sync was started.
> 
> Inside the `<working_directory>/sync-tasks` folder Terraform will create a 
> workspace for each task defined in the configuration.

Reviewer: ```suggestion
 
The folder will be created in the directory where Consul-Terraform-Sync was started and represents the `working_directory` for Terraform.

Inside the `sync-tasks` folder Terraform will create a 
workspace for each task defined in the configuration.
```
